src/Entities/IdToken.php

```diff
@@ -46,14 +46,15 @@ public function convertToJWT(CryptKey $privateKey)
             ->permittedFor($this->getAudience())
             ->expiresAt($this->getExpiration())
             ->issuedAt($this->getIat())
+            ->identifiedBy("123")
```

Comment:
Hi @arietimmerman, was this `123` string expected ? or is it a forgotten debug stuff ?

Reply:
Hmm I intended adding the invocation of the `identifiedBy` method, but for sure I was not intending to add a static parameter. Probably I can change this to let it generate a random identifier each time, but will look at this.

Reply:
```suggestion
            ->identifiedBy(bin2hex(openssl_random_pseudo_bytes(16)))
```

@arietimmerman This might be a good candidate for `jti` generation.
It generates fixed-length hexadecimal strings such as `dfd71db7058c80a87308723474acf8a4`.